operator/install/kube/wasme/crds/wasme.io_v1_crds.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    helm.sh/hook: crd-install
```

Comment:
This is a helm 2 concept, does not work with helm 3, is that intentional? I know we spoke about this briefly in the autopilot PR

Reply:
it is not required for helm 3, but i like leaving the helm 2 option available for now, as not everyone has made the switch.

for now, it isn't important as the helm chart is not currently released. it is only a way to generate the default installation manifest.

i would prefer to iterate on the helm chart when we are ready to begin publishing it

Reply:
But we only have the crds in the `crds` folder, so it's not helm 2 compatible anyway. With new projects I am hesitant to support both. It is quite a bit of work, and not sure how much value it brings.
For now though I think it is fine until the feature is finished in autopilot

Reply:
if that's the case then i agree with removing it, but also agree that the change should happen in autopilot and we'll pick it up here eventually